config/README.md

```diff
@@ -0,0 +1,11 @@
+# Kubernetes Prow Job Configs
```

Comment:
I think this dir should be kubernetes project configs for the tools in general (with subdirs), it looks like that's what is happening but not what this doc says.

Reply:
not tools in general right?

Reply:
ok so in the future we'll move things like gubernator, velodrome deployment files here?

Reply:
yeah non-portable configs ought to be siloed away from the portable/usable code

Reply:
ok edited this one, and added prowjobs/README.md